src/utils/untrancateJson.test.ts

```diff
@@ -0,0 +1,186 @@
+import untruncateJson from './untrancateJson';
+
+describe('untruncateJson', () => {
+  it('returns unmodified valid string', () => {
+    expectUnchanged('"Hello"');
+  });
+
+  it('returns unmodified valid string with bracket characters', () => {
+    expectUnchanged('"}{]["');
+  });
+
+  it('returns unmodified valid string with escaped quotes', () => {
+    expectUnchanged('"\\"Dr.\\" Leo Spaceman"');
+  });
+
+  it('returns unmodified valid string with Unicode escapes', () => {
+    expectUnchanged('ab\\u0065cd');
+  });
+
+  it('returns unmodified valid number', () => {
+    expectUnchanged('20');
+  });
+
+  it('returns unmodified valid boolean', () => {
+    expectUnchanged('true');
+    expectUnchanged('false');
+  });
+
+  it('returns unmodified valid null', () => {
+    expectUnchanged('null');
+  });
+
+  it('returns unmodified valid array', () => {
+    expectUnchanged('[]');
+    expectUnchanged('["a", "b", "c"]');
+    expectUnchanged('[ 1, 2, 3 ]');
+  });
+
+  it('returns unmodified valid object', () => {
+    expectUnchanged('{}');
+    expectUnchanged('{"foo": "bar"}');
+    expectUnchanged('{ "foo": 2 }');
+  });
+
+  it('returns unmodified compound object', () => {
+    expectUnchanged(
+      JSON.stringify({
+        s: 'Hello',
+        num: 10,
+        b: true,
+        nul: 'null',
+        o: { s: 'Hello2', num: 11 },
+        a: ['Hello', 10, { s: 'Hello3' }],
+      })
+    );
+  });
+
+  it('adds a missing close quote', () => {
+    expect(untruncateJson('"Hello')).toBe('"Hello"');
+  });
+
+  it('cuts off trailing "\\" in a string', () => {
+    expect(untruncateJson('"Hello\\')).toBe('"Hello"');
+  });
+
+  it('cuts off a partial Unicode escape in a string', () => {
+    expect(untruncateJson('"ab\\u006')).toBe('"ab"');
+  });
+
+  it('adds "0" to a number cut off at a negative sign', () => {
+    expect(untruncateJson('-')).toBe('-0');
+  });
```

Comment:
very odd

Reply:
what is prefered?

Reply:
If you want to keep these tests than nothing :(